RELEASE.md

```diff
@@ -0,0 +1,40 @@
+# Steps for Preparing a Release
+
+1. Merge any relevant pull requests into the `main` branch.
+2. Pull down the latest version of the `main` branch.
+
+   `git pull origin main`
+
+3. Update `CHANGELOG.md` with the current date, new gem version, and release notes.
+4. Update the `Installation` section of `README.md` to list the new gem version.
+5. Update the value of `s.version` in `pundit-matchers.gemspec` to use the new gem version.
+6. Run the `bundle` command.
+7. Commit the release changes to the `main` branch, e.g:
+
+   `git commit -am "Release version 1.0.0"`
+
+8. Push the changes up to GitHub:
+
+   `git push origin main`
+   
+9. Verify that all CI jobs pass.
+
+10. Build the gem:
+
+    `gem build pundit-matchers.gemspec`
+   
+11. Publish the gem to rubygems.org, e.g:
+
+    `gem push pundit-matchers-1.0.0.gem`
+   
+12. Create a tag for the release, prefixed with a `v`, e.g:
+
+    `git tag v1.0.0`
+
+13. Push the tag to GitHub:
+
+    `git push origin --tags`
```

Comment:
automatized by `rake release` (please check the main comment)

Reply:
I used `rake release` for the last two releases and it has worked well. I'll update the instructions to include this.